azure-mgmt-eventhub/tests/test_azure_mgmt_eventhub.py

```diff
@@ -150,16 +148,13 @@ def test_eh_eventhub_curd(self, resource_group, location):
         # Delete the created eventhub
         geteventhubresponse = self.eventhub_client.event_hubs.delete(resource_group_name, namespace_name, eventhub_name)
 
-        # Delete the create namespace
-        deletenamespace = self.eventhub_client.namespaces.delete(resource_group_name, namespace_name, None, True)
+        time.sleep(60)
 
-        # to verify the deletion of namespace
+        # Delete the create namespace
         try:
-            while(self.eventhub_client.namespaces.get(resource_group_name, namespace_name, None, True).response.status_code != 404):
-                time.sleep(30)
-                continue
-        except ErrorResponseException as ErrorResponse:
-            self.assertEqual(ErrorResponse.message, "Operation returned an invalid status code 'Not Found'")
+            deletenamespace = self.eventhub_client.namespaces.delete(resource_group_name, namespace_name).result()
```

Comment:
At this point of the test, do you expect a fail, or do you expect the delete to succeed?
This is unclear here, since if the delete is a success, you don't test anything. If there an exception, you assert because it was not found, but your scenario should be one of the two but not both at the same time.

Reply:
we have a bug for delete long running operation, where after deleting the namespace, 404 is returned which raises exception. team is working on fixing it. so used assert here.

Reply:
please suggest on this.

Reply:
Team is fixing the RestAPI or the Swagger?